jumpscale/sals/reservation_chatflow/solutions.py

```diff
@@ -575,6 +588,7 @@ def meta_filter(metadata):
                             "IPv4 Address": c_dict["ipv4"],
                             "IPv6 Address": c_dict["ipv6"],
                             "Node": c_dict["node"],
+                            "Farm": self.get_node_farm(c_dict["node"]),
```

Comment:
why we still use this "Titled" case as dict keys? If it's just for consistency with other keys, just ignore my comment :smile: